src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -10077,30 +10149,23 @@ public class AddressClientTest {
       .setUser(user)
       .setStatus(status)
       .build();
-    mockAddresses.addResponse(expectedResponse);
+    mockService.addResponse(expectedResponse);
 
     AddressName address = AddressName.of("[PROJECT]", "[REGION]", "[ADDRESS]");
 
     Operation actualResponse =
         client.deleteAddress(address);
     Assert.assertEquals(expectedResponse, actualResponse);
 
-    List<GeneratedMessageV3> actualRequests = mockAddresses.getRequests();
+    List<String> actualRequests = mockService.getRequestPaths();
     Assert.assertEquals(1, actualRequests.size());
```

Comment:
Why is there no comparison of the request content, or check for the header being sent?

Reply:
I hadn't gotten that far in writing the unit tests. 
Put in TODOs in the http_unit_test.snip for both of those assertion checks.